app/controllers/kms/application_controller.rb

```diff
@@ -1,11 +1,13 @@
 module Kms
   class ApplicationController < ActionController::Base
+    layout 'kms/devise'
```

Comment:
tmw I will add some data from rails 5 so we will be able to work it out exactly, and also that will show why I did change it